.github/workflows/ci.yml

```diff
@@ -17,8 +17,10 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
 
-      - name: Install npm@7
-        run: npm install -g npm@7
+      # Node 16 still ships with npm@8 so for compatibility reasons we're upping this for everything else.
+      - name: Install npm@8
+        if: matrix.node-version != '16.x'
+        run: npm install -g npm@8
```

Comment:
Actually wait what if we just remove this step entirely? Don't we want to run CI on every major Node version with its corresponding NPM version?

Reply:
npm@6 throws a warning if you're installing a library with it that has the npm@7 lockfile format, but maybe it'll be ok? I'll remove this and see.

Reply:
Ah right, they introduced autoinstallation of peerDependencies with npm@7 so with npm@7 on Node 12 and 14 `oas` isn't being installed.

Reply:
I'm going to leave this npm@8 install in so I don't need to manually specify `npm install oas@17.1` in the CI workflow. Anyone installing this library with Node 12 or 14 will get a warning on install that there's an unmet peerDep.